charts/accurate/templates/deployment.yaml

```diff
@@ -0,0 +1,82 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "accurate.fullname" . }}-controller-manager
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app.kubernetes.io/component: controller
+    {{- include "accurate.labels" . | nindent 4 }}
+spec:
+  replicas: {{ .Values.controller.replicas }}
+  selector:
+    matchLabels:
+      app.kubernetes.io/component: controller
+      app.kubernetes.io/name: accurate
+  template:
+    metadata:
+      annotations:
+        checksum/config: {{ include (print $.Template.BasePath "/configmap.yaml") . | sha256sum }}
+      labels:
+        app.kubernetes.io/component: controller
+        app.kubernetes.io/name: accurate
+    spec:
+      containers:
+        - name: manager
+          image: "{{ .Values.image.repository }}:{{ default .Chart.AppVersion .Values.image.tag }}"
+          {{- with .Values.image.pullPolicy }}
```

Comment:
What happens when pullPolicy is not set?  Is it always set?
If so, can we simplify this as below?

```yaml
          imagePullPolicy: {{ .Values.image.pullPolicy }}
```

Reply:
The default is undefined and the imagePullPolicy field is not added if not specified.
https://github.com/cybozu-go/accurate/blob/d5cdc021375e879ca8b6d13811c9b80fffb6c90f/charts/accurate/values.yaml#L10

I will explicitly set IfNotPresent.

Reply:
So, if pullPolicy is not set, this would render as follows, right?

```yaml
spec:
  containers:
  - name: manager
    image: ...
    imagePullPolicy:   # empty
```

I don't like this.  Could you omit `imagePullPolicy:` altogether when pullPolicy is not set?